test/evergreen.yml

```diff
@@ -1990,6 +1990,47 @@ tasks:
           # No of processes to run in the background
           no_of_procs: 10
 
+  - name: format-stress-ppc-zseries-test
+    # Set 25 hours timeout
+    exec_timeout_secs: 90000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger with builtins"
+      - func: "format test script"
+        vars:
+          test_env_vars: ulimit -c unlimited
```

Comment:
Does this work? This doesn't look like a env variable to me. If it works, can we use another variable instead of test_env_vars?

Reply:
Added a new variable "format_test_setting" .